subprojects/packagefiles/sdl2_image/meson.build

```diff
@@ -2,12 +2,87 @@ project(
   'sdl2-image',
   'c',
   version: '2.6.3',
-  meson_version: '>=0.49.0',
+  meson_version: '>=0.59.0',
 )
 
-feature_args = ['-DLOAD_BMP']
+feature_args = []
 
-options = {
+if host_machine.system() == 'windows' and get_option('default_library') != 'static'
+    feature_args += '-DDLL_EXPORT'
+endif
```

Comment:
From the looks of it - dynamic linking on windows probably never worked in the first place. This part likely needs to be ported to sdl2_mixer and sdl2_ttf wraps.